package.json

```diff
@@ -13,9 +13,8 @@
     "npm": "4.2.0"
   },
   "scripts": {
-    "start": "node node_modules/grunt-cli/bin/grunt",
-    "test": "node node_modules/grunt-cli/bin/grunt test",
-    "postinstall": "node node_modules/grunt-cli/bin/grunt install"
+    "start": "gulp",
+    "test": "gulp mocha"
```

Comment:
`gulp mocha` doesn't seem to do anything. Just remove it. I saw @Ewanjiru has an istanbul version for npm test.